### DISCUSSION TITLE
Space ROS Technical Meeting Agenda 2024-09-19

### DISCUSSION THREAD

**Discussion by Bckempa** [space-ros/space-ros#213] [General]
Previous Agenda: https://github.com/space-ros/space-ros/discussions/204
Active Milestone: [`jazzy-2024.10.0`](https://github.com/orgs/space-ros/projects/3/views/11)

### Agenda
- [x] Introduce any new attendees.
- [x] Summarize any PRs merged since last meeting.
  - https://github.com/space-ros/check-commit-message-action/pull/2
  - https://github.com/space-ros/.github/pull/12
  - https://github.com/space-ros/.github/pull/16
  - https://github.com/space-ros/space-ros/pull/202
  - https://github.com/space-ros/space-ros/pull/209
  - https://github.com/space-ros/space-ros/pull/212
  - https://github.com/space-ros/simulation/pull/27
  - https://github.com/space-ros/simulation/pull/29
  - https://github.com/space-ros/docker/pull/186
  - https://github.com/space-ros/demos/pull/37
  - https://github.com/space-ros/demos/pull/53
- [x] Review release follow-up issues and PRs
- [x] Nominate work for next release
- [ ] Open next meeting discussion

**Comment by Bckempa** [comment]
We need to request @gbiggs delete the https://github.com/space-ros/ikos fork now that we're running from the upstream.

**Comment by gbiggs** [comment]
Sure, I can get that done any time. Just let me know when you're ready for it to happen.

**Reply by Bckempa** [reply]
Confirmed during the meeting that we're ready for this. I'll send you an email as well, thanks!

**Comment by franklinselva** [comment]
I would like to add a discussion on #186.

**Reply by Bckempa** [reply]
@franklinselva is going to rebase/freshen-up this PR, reviewers will build and run demos to verify changes, then we'll split the PR for topical merges. @eholum and @Bckempa agreed to make some time for build testing.

**Comment by franklinselva** [comment]
One general question from the last meeting to be carried down is,

Is Space ROS ready with all the features planned for the Humble distro? Are we ready to transition the work to Jazzy?

**Reply by asimonov** [reply]
Jazzy build is kind of working ok, but needs a few clean-ups. The current status: https://github.com/space-ros/space-ros/issues/157#issuecomment-2360415331

**Comment by Bckempa** [comment]
Next meeting agenda: https://github.com/space-ros/space-ros/discussions/214
